### ISSUE TITLE
Machine learning engineer: Prepare data set with ground truth mask for watershed transform

### ISSUE THREAD

**Issue by krook** [Code-and-Response/ISAC-SIMO#62]
We need to prepare the training data set for the deep watershed transform approach

These are the proposed steps. 
- Generate the segmented image with the mask (with GIMP/Glimpse)
- Then process the images against a model
- Then process the centroids on those images
- Then use that to calculate alignment quality

What additional steps might be taken to get to the images we need
